<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogCreateNewDictionary</class>
 <widget class="QDialog" name="DialogCreateNewDictionary">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>470</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>430</y>
     <width>161</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>17</x>
     <y>18</y>
     <width>361</width>
     <height>401</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>10</number>
    </property>
    <property name="sizeConstraint">
     <enum>QLayout::SetDefaultConstraint</enum>
    </property>
    <item>
     <widget class="QLabel" name="label_dict_filename">
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Имя файла для нового словаря, латиницей</string>
      </property>
      <property name="textFormat">
       <enum>Qt::PlainText</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="dict_filename">
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_dict_name">
      <property name="text">
       <string>Название словаря</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="dict_name"/>
    </item>
    <item>
     <widget class="QLabel" name="label_dict_author">
      <property name="text">
       <string>Имя составителя словаря</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="dict_author"/>
    </item>
    <item>
     <widget class="QLabel" name="label_dict_credits">
      <property name="text">
       <string>Имена соавторов словаря</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="dict_credits"/>
    </item>
    <item>
     <widget class="QLabel" name="label_dict_tags">
      <property name="text">
       <string>Принадлежность языка к группам, через запятую</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="dict_tags"/>
    </item>
    <item>
     <widget class="QLabel" name="label_dict_description">
      <property name="text">
       <string>Краткое описание словаря</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="dict_description"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogCreateNewDictionary</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogCreateNewDictionary</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
